namespaces-disallowlist.yaml

```
@@ -10,8 +10,8 @@
 - mlhackweek_search
 - mozphab
 - mozregression
-- rally_core
-- rally_markup_fb_pixel_hunt
-- rally_study_zero_one
-- reference_browser
 - pine
+- pioneer-*
```

Looks like namespaces are normally in snake_case, so this probably needs to be:
```suggestion
- pioneer_*
```